app/src/organisms/InterventionModal/StackerEmptyInterventionContent.tsx

```diff
@@ -0,0 +1,107 @@
+import { ComponentProps } from 'react'
```

Comment:
```suggestion
import { ComponentProps } from 'react'
```
and move to the type import section!